qbft/controller.go

```diff
@@ -82,10 +79,10 @@ func (c *Controller) ProcessMsg(msg *SignedMessage) (*SignedMessage, error) {
 	if IsDecidedMsg(c.Share, msg) {
 		return c.UponDecided(msg)
 	} else if c.isFutureMessage(msg) {
-		return c.UponFutureMsg(msg)
-	} else {
-		return c.UponExistingInstanceMsg(msg)
+		return nil, fmt.Errorf("future msg from height %d, could not process", msg.Message.Height)
```

Comment:
I tend to like less errors with variables in them.. up to you but lets use one format and not a mix

Reply:
done